okhttp/src/jvmTest/java/okhttp3/internal/tls/CertificatePinnerChainValidationTest.java

```diff
@@ -182,7 +184,9 @@ public void setup(MockWebServer server) {
     // https://github.com/square/okhttp/issues/4729
     platform.expectFailureOnConscryptPlatform();
     platform.expectFailureOnCorrettoPlatform();
-    platform.expectFailureFromJdkVersion(19);
+
+    // unpredictable as to whether it passes or fails? Environmental?
+    Assumptions.assumeTrue(PlatformVersion.INSTANCE.getMajorVersion() < 19);
```

Comment:
What happens if you remove this completely? Perhaps JDK 19 just changed?

Reply:
I'll try. But flipped in a day or so.